studio/__main_unit__.py

```diff
@@ -118,11 +135,37 @@ async def index(request: Request):
     return await root(request)
 
 
+async def check_load_balancer() -> Dict[str, Any]:
```

Comment:
I would like to reiterate the need for this functionality.

*If this function is needed for AWS environment checks, it would seem that it would be more appropriate to prepare it in the AWS infrastructures or in could_startup.sh, rather than in the internal code of optinist.